src/rdkafka_broker.c

```diff
@@ -596,17 +596,18 @@ rd_kafka_broker_send (rd_kafka_broker_t *rkb, rd_slice_t *slice) {
 static int rd_kafka_broker_resolve (rd_kafka_broker_t *rkb) {
 	const char *errstr;
 
+	rd_ts_t now = rd_clock();
```

Comment:
No need for temp variable here, only used once.

Reply:
If that’s your preference. As someone who recently spent a lot of time reading librdkafka from a place of relative ignorance, more temporary variables would have been very helpful.